Vic2ToHoI4/Data_Files/blankMod/output/events/ReconquestWarsEvents.txt

```diff
@@ -0,0 +1,130 @@
+add_namespace = reconquest_wars
+
+# Territory or war Ultimatum
+country_event = {
+	id = reconquest_wars.1
+	title = reconquest_wars.1.t
+	desc = reconquest_wars.1.d
+	picture = GFX_report_event_german_troops
+	
+	is_triggered_only = yes
+	
+	option = {
+		name = reconquest_wars.1.a
+		FROM = {
+			country_event = { id = reconquest_wars.5 }
```

Comment:
Is this correct? I only see events going up to 4.

Reply:
> Is this correct? I only see events going up to 4.

Good catch. Consequences of splitting 6 neighbour_wars and renaming half to reconquest_wars but not changing ids